woocommerce/woocommerce-gateway-stripe
# Add conditional before trying to add mask to CPF/CNPJ field

Fixes #2233

## Changes proposed in this Pull Request:
Add a conditional before trying to add a mask to CPF/CNPJ field to prevent errors when Boleto is not Active.

I could not simulate an error during development but customers have had bad experiences on their end.

## Testing instructions
- Try to checkout with Boleto active(you'll need to set country to brazil inside your stripe [dashboad](https://dashboard.stripe.com/settings/account))
- Make sure the CPF/CNPJ mask is applied (52321590033 should become 523.215.900-33)
- Make sure no errors show up on the console

- Try to checkout with boleto inactive
- No errors should show up on the console


**Post merge**
